docker/cygnus-ngsi/Dockerfile

```diff
@@ -116,13 +116,13 @@ RUN \
     echo "INFO: Create Cygnus log folder" && \
     mkdir -p /var/log/cygnus && \
     echo "INFO: Cleanup to thin the final image... doing optimizations..." && \
-    echo "INFO: Java runtime not needs JAVA_HOME... Unsetting..." && \
-    unset JAVA_HOME && \
     cd ${CYGNUS_HOME}/cygnus-common && \
     ${MVN_HOME}/bin/mvn ${MAVEN_ARGS} clean && \
     cd ${CYGNUS_HOME}/cygnus-ngsi && \
     ${MVN_HOME}/bin/mvn ${MAVEN_ARGS} clean && \
     rm -rf /root/.m2 && rm -rf ${MVN_HOME} && rm -rf ${FLUME_HOME}/docs && rm -rf ${CYGNUS_HOME}/doc && rm -f /*.tar.gz && \
+    echo "INFO: Java runtime not needs JAVA_HOME... Unsetting..." && \
+    unset JAVA_HOME && \
```

Comment:
Note that the other two Dockerfile don't unset JAVA_HOME, thus this modification is not needed in these Dockerfiles.

NTC